canonical-websites/vanillaframework.io
# Design review tweaks

## Done
Completed the list of design updates.

## QA
- Pull down the code and run `npm run develop`
- Check each todo in the [related issue](https://github.com/canonical-websites/vanillaframework.io/issues/11)

## Details
Fixes https://github.com/canonical-websites/vanillaframework.io/issues/11
